apps/gateway/lib/gateway/router.ex

```diff
@@ -34,8 +34,13 @@ defmodule Gateway.Router do
       put "/currency", CurrencyController, :modify_currency
       get "/claim_daily_reward", UserController, :claim_daily_reward
       get "/get_daily_reward_status", UserController, :get_daily_reward_status
-      get "/quest/:quest_id/reroll_daily_quest", QuestController, :reroll_daily_quest
-      get "/quest/:quest_id/complete_bounty", QuestController, :complete_bounty
+
+      scope "/quest" do
+        put "/add_quests", QuestController, :add_quests
```

Comment:
The semantics of this endpoints are kinda weird, a PUT under `/quests` you would expect it to be modifying a quest, but it is actually modifying a user

Reply:
Maybe replacing this with a post instead of a put may do the trick